### DISCUSSION TITLE
Appcopier - Initial release

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Appcopier#2] [Announcements]
This small project is still in the making. It allows you to back up and restore your most important Windows 11 preferences and settings offline and locally. The app [mimics the new backup app of Windows 11](https://support.microsoft.com/en-us/windows/back-up-your-windows-pc-87a81f8a-78fa-456e-b521-ac0560e32338) -  which is part of the Windows 11 2023 Update (23H2) - but without the obligation of the cloud. I will certainly expand and enhance it over time. 

How it works? Quite simple! Only registry entries and/or associated folders and files of the respective area are exported. This process is very quick and lightweight, akin to the weight of a fly. So don't be surprised if the first backups fly through in the nanosecond range. For the future, I could envision an addition of a more dynamic option in the form of scripts/plugins, where even more complex things could be backed up.

![screenshot](https://github.com/builtbybel/Appcopier/blob/main/assets/appcopier.png?raw=true)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Appcopier/releases/tag/0.12'>Appcopier - Initial release</a>.</em>

**Comment by deus-e-x** [comment]
Hi Belim,

what a great idea for an app. It will surely come in handy for many users in the foreseeable future.

 I'd like to suggest to add backup of the Windows network settings, and even better if possible, an option to backup/restore the entire registry.

**Reply by Belim** [reply]
You got it ;) Network Settings sounds cool and legitimate. I've added it in the latest build 0.12.30

**Comment by ed-br** [comment]
A very cool app, not like the frustrating Microsoft "OneDrive" Backup App

**Comment by katastrophe92** [comment]
Hi @Belim ! Thanks for the amazing project! I like it a lot. I want it to see it grow and include all type of Apps, settings, preferences, everything.

I do backup my apps by syncing the AppData folder and after I install a fresh windows first I install my apps through [WingetUI](https://github.com/marticliment/WingetUI) restore file, then copy over the already backup AppData folder. But this is way complicated than you're project.

**Comment by upintheairsheep** [comment]
Where’s the code for the app itself? I’ve searched for strings but couldn’t find them.

**Comment by emorrank01** [comment]
Hi Belim,

A very useful application. Have you considered adding a system backpoint to the application?

**Comment by TechX-xx** [comment]
I really like this app but would like to be able to set a specific location for the backups. Would that be possible?
